### PR TITLE
feat: tabs and swaps components events migration

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**

tabs and swaps components under UI folder, with new Segment metrics.

Snapshot updated on Tabs, needs to migrate to react-testing-library
## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

https://recordit.co/w5SwTPfpHj
https://recordit.co/n7sVQDCkCL

** Not all the events were recorded **
## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [ ] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by Cal-L — DISMISSED** [review]
LGTM

File: app/components/UI/Swaps/components/TokenSelectModal.js [hunk 1]
```diff
@@ -302,17 +303,15 @@ function TokenSelectModal({
   const handlePressImportToken = useCallback(
     (item) => {
       const { address, symbol } = item;
-      InteractionManager.runAfterInteractions(() => {
```
Comment (NicolasMassart) [inline app/components/UI/Swaps/components/TokenSelectModal.js:305]: Should not be an annonymous event

Comment (tommasini) [reply app/components/UI/Swaps/components/TokenSelectModal.js:305]: Good catch!!

**Review by NicolasMassart — DISMISSED** [review]
Looks good to me.

**Review by NicolasMassart — DISMISSED** [review]

**Review by NicolasMassart — APPROVED** [review]
Looks good to me.

**Pull request merged**